tensorflow/lite/experimental/micro/examples/micro_speech/CMSIS/hanning.cc

```diff
@@ -0,0 +1,67 @@
+/* SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we switch to the standard Copyright TensorFlow Authors... form for the license? Thanks!